[IMP] sales: updated import products doc for 16 (#5861)
odoo/documentation: content/applications/sales/sales/products_prices/products/import.rst

editable directly on the product form, they can also be modified via a product import.

As mentioned, relation fields of this nature can **only** be imported for products if they already
exist in the database. For example, if a user attempts to import a product with a *Product Type*, it
can only be one of the preconfigured product types existing in the database (e.g. *Storable
Product*, *Consumable*, etc.).

To import information for a relation field on a product import template spreadsheet, add the name of
the field as a column name/title on the spreadsheet. Then, on the appropriate product line, add the
desired relation field option.

When all desired relation field information has been entered, save the spreadsheet, and import it
to the database, per the process mentioned above (:menuselection:`Sales app --> Products -->
Products --> ⚙️ (gear) icon --> Import records --> Upload File`).

Once the spreadsheet with the newly-configured relation field information has been uploaded, click
:guilabel:`Import`, and Odoo returns to the :guilabel:`Products` page.

When the newly-changed/modified products (complete with the new relation field information) has been

> imported and uploaded, that new information can be found on that product's product page.

Reviewer: ```suggestion
imported and uploaded, that new information can be found on the :guilabel:`Products` page.
```
I understand you are trying to refer to an individual product record (page) here, so I do not know if this is the best wording, but I don't think "product's product page" works. Maybe just "Product record"?

Attributes and values
---------------------

Odoo also allows users to import product attributes and values that can be used for products that
already exist in the database and/or with imported products.

> To import attributes and values, a separate Excel or CSV file dedicated to attributes and values

Reviewer: ```suggestion
To import attributes and values, a separate spreadsheet or CSV file dedicated to attributes and values
```

There can be other columns, as well, but these columns are **required** in order to properly import
products with specific variants.

.. image:: import/product-attribute-spreadsheet-import.png
   :align: center
   :alt: Product variants spreadsheet with product attributes and variants for import purposes.

- :guilabel:`Name`: product name

- :guilabel:`Product Attributes / Attribute`: name of attribute

- :guilabel:`Product Attributes / Values`: values pertaining to the corresponding attribute

.. tip::
   To import multiple values, separate them by *just* a comma (**not** a comma followed by a space)
   in the product import template spreadsheet.

When the desired products and product variants have been entered and saved in the spreadsheet, it's
time to import and upload them into Odoo. To do that, navigate to :menuselection:`Sales app -->
Products --> Products --> ⚙️ (gear) icon --> Import records --> Upload File`.

Once the spreadsheet with the newly-configured products, and product variants, has been uploaded,
click :guilabel:`Import`, and Odoo returns to the :guilabel:`Products` page. That's where the
newly-added products can be found.

> To view and modify the product variants on any products, select the desired product from the
> :guilabel:`Products` page, and click the :guilabel:`Attributes & Variants` tab. That's where the
> product attributes and variants can be found and modified, if necessary.

Reviewer: ```suggestion
To view and modify the attributes and variants on any products, select the desired product from the
:guilabel:`Products` page, and click the :guilabel:`Attributes & Variants` tab.
```
